.github/workflows/test-workflow.yml

```
@@ -0,0 +1,47 @@
+name: Test workflow
+
+on:
+  workflow_call:
+    inputs:
+      node-version:
+        required: true
+        type: number
+
+jobs:
+  test:
+    name: '${{ matrix.package }} - node.js v${{ inputs.node-version }}'
```

[Docs] Nit, this is what we call it everywhere:

```suggestion
    name: '${{ matrix.package }} - Node.js v${{ inputs.node-version }}'
```